data_rentgen/consumer/settings/consumer.py

```diff
@@ -0,0 +1,215 @@
+# SPDX-FileCopyrightText: 2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+
+import textwrap
+from typing import Literal
+
+from pydantic import BaseModel, ByteSize, Field, model_validator
+
+
+class ConsumerSettings(BaseModel):
+    """Data.Rentgen consumer-specific settings.
+
+    These options are passed directly to
+    `AIOKafkaConsumer <https://aiokafka.readthedocs.io/en/stable/api.html#aiokafka.AIOKafkaConsumer>`_.
+
+    Examples
+    --------
+
+    .. code-block:: bash
+
+        DATA_RENTGEN__CONSUMER__TOPICS_LIST=["input.runs"]
+        DATA_RENTGEN__CONSUMER__GROUP_ID=data-rentgen
+        DATA_RENTGEN__CONSUMER__FETCH_MAX_WAIT_MS=5000
+        DATA_RENTGEN__CONSUMER__MAX_PARTITION_FETCH_BYTES=5MiB
```

Comment:
Does it support parsing?

Reply:
Yep https://docs.pydantic.dev/1.10/usage/types/#bytesize